rediscluster/clu.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```

Comment:
As far as I can see it is only working with python3, since at least on debian you need backported `python3-redis` (2.10.5-2~bpo8+1) to get it running.